### PR TITLE
chore(template v3): add sso tab templates

### PR DESCRIPTION
**Pull request by hund030**

### CODE REVIEW

File: templates/scenarios/sso-tab-js/.fx/configs/config.dev.json.tpl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (huimiu) [inline templates/scenarios/sso-tab-js/.fx/configs/config.dev.json.tpl:1]: Just curious, is the tpl file a new design in V3? I don't seem to see it in the current template

Comment (hund030) [reply templates/scenarios/sso-tab-js/.fx/configs/config.dev.json.tpl:1]: No. We add .tpl suffix to identify which file includes placeholders. You can find such files in v2 templates, too.
https://github.com/OfficeDev/TeamsFx/blob/dev/templates/tab/csharp/default/Interop/InteropModuleBase.cs.tpl

Comment (huimiu) [reply templates/scenarios/sso-tab-js/.fx/configs/config.dev.json.tpl:1]: OK, I got it. It looks like most of it is used in csharp's template

File: templates/scenarios/sso-tab-js/templates/azure/main.bicep [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@secure()
+param provisionParameters object
```
Comment (huimiu) [inline templates/scenarios/sso-tab-js/templates/azure/main.bicep:2]: Is this variable defined repeatedly? I guess you want to put all the variable definitions in the` config.bicep` file, I don't know if I understand it correctly.

Comment (hund030) [reply templates/scenarios/sso-tab-js/templates/azure/main.bicep:2]: I just copy the biceps from current teamsfx project. Here the config.bicep is useless, but I don't want to make the change in this PR.

**Review by huimiu — APPROVED** [review]

**Pull request merged**